### PR TITLE
Testing workflows

### PR DESCRIPTION
**Pull request by Bailonis**
This pr is to test the workflows, for example the deployment to docker hub

### CODE REVIEW

File: .github/workflows/dash-dependency-check.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Generate dependencies file
+        run: cd backend && mvn dependency:list -DskipTests -Dmaven.javadoc.skip=true -DappendOutput=true -DoutputFile=maven.dependencies
```
Comment (carslen) [inline .github/workflows/dash-dependency-check.yml:62]: Repo doesn't contain a `backend` directory but `demand-capacity-mgmt-backend` so `cd backend` seems to be wrong.

Creating the dependency list fails:

```
*[feat/workflows][~/projects/cat-x/repos/tx-demand-capacity-mgmt/demand-capacity-mgmt-backend]$ mvn dependency:list -DskipTests -Dmaven.javadoc.skip=true 
-DappendOutput=true -DoutputFile=maven.dependencies
[...]
Downloaded from central: https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-actuator-autoconfigure/2.7.8/spring-boot-actuator-autoconfigure-2.7.8.jar (571 kB at 359 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/io/micrometer/micrometer-core/1.9.7/micrometer-core-1.9.7.jar (659 kB at 398 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/mockito/mockito-core/4.5.1/mockito-core-4.5.1.jar (662 kB at 397 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-actuator/2.7.8/spring-boot-actuator-2.7.8.jar (687 kB at 404 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/assertj/assertj-core/3.22.0/assertj-core-3.22.0.jar (5.2 MB at 2.9 MB/s)
Downloading from spring-snapshots: https://repo.spring.io/snapshot/org/eclipse/tractusx/demand-capacity-mgmt-specification/demand-capacity-mgmt-specification/1.0-SNAPSHOT/demand-capacity-mgmt-specification-1.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20.658 s
[INFO] Finished at: 2023-10-20T11:46:18+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project demand-capacity-mgmt-backend: Could not resolve dependencies for project org.eclipse.tractusx.demand-capacity-mgmt-backend:demand-capacity-mgmt-backend:jar:0.0.1-SNAPSHOT: The following artifacts could not be resolved: org.eclipse.tractusx.demand-capacity-mgmt-specification:demand-capacity-mgmt-specification:jar:1.0-SNAPSHOT (absent): Could not find artifact org.eclipse.tractusx.demand-capacity-mgmt-specification:demand-capacity-mgmt-specification:jar:1.0-SNAPSHOT in spring-snapshots (https://repo.spring.io/snapshot) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Consider using maven plugin instead:
- https://github.com/eclipse/dash-licenses#example-maven-plugin
- https://blog.waynebeaton.ca/posts/ip/dash-license-tool-maven-plugin/

**Review by carslen — CHANGES_REQUESTED** [review]
At least `DEPENDENCIES_FRONTEND` is not up to date. Please check workflow for backend creating `DEPENDENCIES_BACKEND` and ensure to update the file in this PR (if required).

File: .github/workflows/dash-dependency-check.yml (hunk 1 above)
Comment (Bailonis) [reply .github/workflows/dash-dependency-check.yml]: Resolved

**Comment by Bailonis** [comment]
@carslen can you take a look, please?

**Comment by SebastianBezold** [comment]
HI @Bailonis,

as discussed, we are closing this PR, so we can properly address the TRG issues, that did arise in the meantime

**Pull request closed without merging**